example/cmd/device-simple/res/configuration.toml

```
@@ -85,6 +85,7 @@ PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name
   [MessageQueue.Topics]
   CommandRequestTopic = "edgex/device/command/request/device-simple/#"   # subscribing for inbound command requests
   CommandResponseTopicPrefix = "edgex/device/command/response"   # publishing outbound command responses; <device-service>/<device-name>/<command-name>/<method> will be added to this publish topic prefix
+  SystemEventTopic = "edgex/system-events/#"
```

Do we want make this more specific to `core-metadata` device system events and ones only for this service?

`/<source>/<type>/<action>/<owner>/<profile> will be added to this Publish Topic `

```suggestion
  SystemEventTopic = "edgex/system-events/core-metadata/device/#/device-simple/#"
```